src/core/server/saved_objects/service/index.ts

```diff
@@ -40,7 +40,10 @@ export interface SavedObjectsService<Request = any> {
   getSavedObjectsRepository(...rest: any[]): any;
   importExport: {
     objectLimit: number;
-    importSavedObjects(options: SavedObjectsImportOptions): Promise<SavedObjectsImportResponse>;
+    importSavedObjects(
+      options: SavedObjectsImportOptions,
+      srcSpaceId?: string
```

Comment:
We don't need this either, the `options.namespace` denotes the target namespace.

Reply:
Yes, but we need the source space id to retrieve the correct title for the objects we are about to overwrite.